core/classes/Core/Util.php

```diff
@@ -283,6 +283,22 @@ public static function replaceAnchorsWithText(string $data): string {
         return URL::replaceAnchorsWithText($data);
     }
 
+    private static function getSettingsCache(?string $module): ?array {
```

Comment:
Could these parameters not just be
```php
private static function getSettingsCache(string $module = 'core'): ?array {
```
to save the assignment on 287 etc

Reply:
I used null because that was consistent with how it's actually stored in the database, but I will change it.